cinder/volume/drivers/lightos.py

```diff
@@ -1137,10 +1162,34 @@ def __remove_volume_acl(
         if not acl:
             acl.append('ALLOW_NONE')
 
+        ip_acl = data.get('IPAcl')
+        if not ip_acl:
+            LOG.warning('Got LightOS volume without IP ACL?! data: %s', data)
+            return False
```

Comment:
This code should only be executed if use_ipacl (suggest you put it in a function that returns immediately if use_ipacl is False)